test/deploy/kubernetes/service.sh

```diff
@@ -87,6 +87,8 @@ fi
 # Create Control RunTime Node (POD)
```

Comment:
Apply message service before creating control pod... Infact even before creating HA node...
Also please handle deletion case for message service

Reply:
Line No. 114 change message as:
`print_header "Creating Storage RunTime Node - $NODE_NAME";`

Reply:
Line No. 88 change message as:
`print_header "Creating Control RunTime Node - Cluster";`

Reply:
Removed the unwanted code